### PR TITLE
Change constant time and cache resistant ECC mulmod

### PR DESCRIPTION
**Pull request by SparkiDev**
Ensure points being operated on change to make constant time.

### CODE REVIEW

File: wolfcrypt/src/wolfmath.c [hunk 1]
```diff
@@ -89,6 +89,30 @@ mp_digit get_digit(mp_int* a, int n)
     return (n >= a->used || n < 0) ? 0 : a->dp[n];
 }
 
+int mp_cond_copy(mp_int* a, int copy, mp_int* b)
```
Comment (toddouska) [inline wolfcrypt/src/wolfmath.c:92]: Looks good but please add a basic comment here describing why and what this does for ease of future maintenance.

Comment (SparkiDev) [reply wolfcrypt/src/wolfmath.c:100]: Comments added.

**Comment by carnil** [comment]
This pull request seem to have been associated with CVE-2020-11713.

**Comment by SparkiDev** [comment]
Yes, this pull request is based on the report.

**Pull request merged**